cachito/web/metrics.py

```diff
@@ -0,0 +1,30 @@
+import os
+
+from prometheus_client import multiprocess
+from prometheus_client.core import CollectorRegistry
+from prometheus_flask_exporter import PrometheusMetrics
+
+
+def init_metrics():
+    """
+    Initialize the Prometheus Flask Exporter.
+
+    :return: a Prometheus Flash Metrics object
+    :rtype: PrometheusMetrics
+    """
+    registry = CollectorRegistry()
+    if "PROMETHEUS_MULTIPROC_DIR" not in os.environ:
+        raise Exception(
+            "Temp directory for prometheus-flask metrics not defined in environment variable: \
+            PROMETHEUS_MULTIPROC_DIR"
+        )
+    else:
+        multiproc_temp_dir = os.environ["PROMETHEUS_MULTIPROC_DIR"]
```

Comment:
Thinking more about it, why this is envvar and not just another attribute in config file?

Reply:
Went one farther and used the Tempfile module to create the temp directory if not specified (to allow tox to provide a location & clean it up at the end of a test run)